src/cluster/placement/options.go

```diff
@@ -68,6 +71,7 @@ type options struct {
 	validateFn          ValidateFn
 	nowFn               clock.NowFn
 	allowPartialReplace bool
+	allowNonValidZones  bool
```

Comment:
Not a fan of the name, perhaps `allowAnyZone` ?

Reply:
'allowAllZones` maybe?

Reply:
Good call.